perllib/FixMyStreet/App/Controller/Open311.pm

```diff
@@ -233,44 +233,42 @@ sub output_requests : Private {
 
         my $request =
         {
-            'service_request_id' => [ $id ],
-            'title' => [ $problem->title ], # Not in Open311 v2
-            'detail'  => [ $problem->detail ], # Not in Open311 v2
-            'description' => [ $problem->title .': ' . $problem->detail ],
-            'lat' => [ $problem->latitude ],
-            'long' => [ $problem->longitude ],
-            'status' => [ $problem->state ],
-#            'status_notes' => [ {} ],
-            'requested_datetime' => [ w3date($problem->confirmed) ],
-            'updated_datetime' => [ w3date($problem->lastupdate) ],
-#            'expected_datetime' => [ {} ],
-#            'address' => [ {} ],
-#            'address_id' => [ {} ],
-            'service_code' => [ $problem->category ],
-            'service_name' => [ $problem->category ],
-#            'service_notice' => [ {} ],
-#            'zipcode' => [ {} ],
-            'interface_used' => [ $problem->service ], # Not in Open311 v2
+            'service_request_id' => $id,
+            'title' => $problem->title, # Not in Open311 v2
+            'detail'  => $problem->detail, # Not in Open311 v2
+            'description' => $problem->title .': ' . $problem->detail,
+            'lat' => $problem->latitude,
+            'long' => $problem->longitude,
+            'status' => $problem->state,
+#            'status_notes' => {},
```

Comment:
Note I haven't commented anything out, I just fixed the arrayrefs in them too :) They are other fields that an Open311 service could/should return, so I think it's good to leave them in.